src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name();
```

Comment:
Calling 'Color.values()' multiple times can be inefficient as it creates a new array each time. Consider storing the result in a local variable at the start of the method to avoid this overhead.